RunFromSourceGuideForMac.md

```diff
@@ -0,0 +1,19 @@
+# Prerequisite Software
+Here is a list of software to install and source code to download.
+1. [Python](https://www.python.org/downloads/macos/), 3.8 or newer is necessary.
+2. [Xcode](https://apps.apple.com/us/app/xcode/id497799835)
+3. [Archipelago Source](https://github.com/ArchipelagoMW/Archipelago/archive/refs/heads/main.zip), this will download the source code in a zipped format.
```

Comment:
Downloading "Source code (zip)" from the latest release from https://github.com/ArchipelagoMW/Archipelago/releases would be recommended to be on a stable version as a user. HEAD is only recommended for development.

Reply:
Good call out, change made.